ansible/roles/software/ckan/catalog/ckan-app/defaults/main.yml

```
@@ -86,3 +86,5 @@ catalog_ckan_fgdc2iso_host: localhost
 ckan_catalog_next: false
 ckan_production_ini_template: etc_ckan_production.ini.j2
 ckan_uses_gunicorn: false
+newrelic_license_key: some-secret
+newrelic_app_name: catalog-app-default
```

Why not just match what is in New Relic?
```suggestion
newrelic_app_name: catalog
```